Python/omevv/v1/scripts/CreateRepoProfile.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import sys
+import time
+import warnings
+from webbrowser import get
+
+import requests,json,base64
+warnings.filterwarnings("ignore")
+
+parser = argparse.ArgumentParser(
+    description="Python script using OME api to create Repository profile"
+);
+parser.add_argument('-ip', help='OMEVV IP address', required=True);
+parser.add_argument('-ome_user', help='OME username', required=True);
+parser.add_argument('-ome_pswd', help='OME password', required=True);
+parser.add_argument('-omevv_user', help='OMEVV username', required=True);
+parser.add_argument('-omevv_pswd', help='OMEVV password', required=True);
+parser.add_argument('-profilename', help='repository profile name', required=True);
+parser.add_argument('-description', help='repository profile description', required=False);
+parser.add_argument('-profileType', help='repository profile type', required=True);
+parser.add_argument('-sharepath', help='sharepath', required=True);
+parser.add_argument('-spcred', help='sharepath credential', required=True);
+parser.add_argument('-protocol',help='protocol type',choices=['CIFS','NFS','HTTP','HTTPS','UNAVAILABLE'],required = True);
+parser.add_argument('-certificateCheck',help ='certificate check required?',required = False);
+
+args = vars(parser.parse_args())
```

Comment:
Let's avoid global variables when possible and pass it to the required methods from the calling ones.
Recommend that only those variables that are not meant as parameters but dictate execution(retries etc) are kept as global.